README.md

```diff
@@ -1 +1,31 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
```

Comment:
In general, the overviews should be a friendly introduction to the topic. Don't include "big words" -  terms and phrases that the student won't be familiar with. Feel free to add a friendly introduction, ie "In this lesson, we'll be discussing AngularJS, an open-source framework released in 2009..." Explain the problem it's solving in term of what the students already know - Rails, Javascript, jQuery, and AJAX.